src/wasm_opt.rs

```diff
@@ -54,7 +54,7 @@ fn download_wasm_opt() -> Result<&'static Path> {
 ///
 /// # Usage
 ///
-/// ```rust,no_run
+/// ```rust,ignore
```

Comment:
I knew you would get confused by this.

`no_run` and `ignore` are actually 2 different flags. You want to avoid using `ignore` **as much as possible** because it easily makes your documentation outdated.

`no_run` should also be avoided but is less dramatic [dramatic music gif] than `ignore` because **at least the code is compiled**. In `ignore`, the code is **not compiled!!**

Then finally you should prefer to use neither. The best code doc is the doc that serves also as test. Any breaking change in the API will also break the doc test. This is an amazing feature of Rust.

I don't know any other language that has that feature but I can tell you that in other languages we usually avoid putting code in doc because those are not tested, probably not working and easily become wrong and misleading.

```suggestion
/// ```rust,no_run
```

Reply:
(And yes: things breaking (at compilation) is a damn amazing feature :heart:)